docs/presentation/content/RelatedPropertiesSpecification.md

```diff
@@ -172,3 +174,38 @@ The attribute allows loading additional related properties that are related to t
 ```
 
 ![Example of using the "nested related properties" attribute](./media/relatedpropertiesspecification-with-nestedrelatedproperties-attribute.png)
+
+### Attribute: `relationshipProperties`
+
+List of names or definitions of relationship class properties that should be included in the content. The properties are taken from the relationship used in the last step of the relationship path specified in the [`propertiesSource`](#attribute-propertiessource). In addition, a couple of special values are allowed:
+
+- `"_none_"` means none of the relationship properties should be picked up.
+- `"*"` means all relationship properties should be picked up.
+
+|                   |                                                             |
+| ----------------- | ----------------------------------------------------------- |
+| **Type**          | `Array<string \| PropertySpecification> \| "_none_" \| "*"` |
+| **Is Required**   | No                                                          |
+| **Default Value** | `[]`                                                        |
+
+```ts
+[[include:Presentation.Content.Customization.RelatedPropertiesSpecification.RelationshipProperties.Ruleset]]
+```
+
+![Example of using the "relationship properties" attribute](./media/relatedpropertiesspecification-with-relationshipproperties-attribute.png)
+
+### Attribute: `forceCreateRelationshipCategory`
+
+Specifies whether a relationship category should be created regardless of whether any relationship properties were included.
+
+|                   |           |
+| ----------------- | --------- |
+| **Type**          | `boolean` |
+| **Is Required**   | No        |
+| **Default Value** | `false`   |
+
+```ts
+[[include:Presentation.Content.Customization.RelatedPropertiesSpecification.ForceCreateRelationshipCategory.Ruleset]]
+```
+
+![Example of using the "force create relationship category" attribute](./media/relatedpropertiesspecification-with-forcecreaterelationshipcategory-attribute.png)
```

Comment:
It would be easier to understand the effect of `forceCreateRelationshipCategory` if there was an another screenshot where it's set to `false`.